### PR TITLE
Normalize copyright headers

### PR DESCRIPTION
**Pull request by dschwen**
Adds missing headers, fixes wrong headers, and fixes minor formatting issues.

Refs #4729.

### CODE REVIEW

File: modules/phase_field/include/kernels/PFFracBulkRate.h [hunk 1]
```diff
@@ -1,17 +1,9 @@
 /****************************************************************/
-/*               DO NOT MODIFY THIS HEADER                      */
```
Comment (dschwen) [inline modules/phase_field/include/kernels/PFFracBulkRate.h]: :fu: :cop:

**Comment by dschwen** [comment]
Ugh, I guess the app test fails until MARMOT Merge Request #168 is merged.

File: modules/phase_field/include/kernels/PFFracBulkRate.h (hunk 1 above)
Comment (permcody) [inline modules/phase_field/include/kernels/PFFracBulkRate.h]: Nicely done...

**Comment by permcody** [comment]
I merged Marmot's change - I'll get these through later today or tomorrow.

**Pull request merged**